lib/core.js

```diff
@@ -10,6 +10,7 @@ import { promiseRetry, bind, uuid } from './util';
  * @typedef {object} CoreConfig
  * @prop {string} [key] The API key ("publishable_key_..." or "pk_...") used to authenticate with the Ravelin API.
  * @prop {string} [api] A string of the format "https://api.ravelin.net" which forms the base of API requests.
+ * @prop {string} [prefix=ravjs-] A string of the default prefix for the deviceId
```

Comment:
Not rjs-, given the body of the function Core (specifically L44)?

Reply:
Whoops!